build_tools/cmake/build_e2e_test_artifacts.sh

```diff
@@ -26,10 +30,53 @@ set -xeuo pipefail
 
 BUILD_DIR="${1:-${IREE_BUILD_E2E_TEST_ARTIFACTS_DIR:-build-e2e-test-artifacts}}"
 IREE_HOST_BIN_DIR="$(realpath ${IREE_HOST_BIN_DIR})"
+BENCHMARK_PRESETS="${IREE_BENCHMARK_PRESETS:-}"
+BUILD_DEFAULT_BENCHMARK_SUITES="${IREE_BUILD_DEFAULT_BENCHMARK_SUITES:-1}"
 
 source build_tools/cmake/setup_build.sh
 source build_tools/cmake/setup_tf_python.sh
 
+declare -a BUILD_TARGETS
```

Comment:
We should probably rewrite this (and like all our build scripts) in Python... (not this PR though)